internal/utils/config/src/main/resources/ditto-mongo.conf

```diff
@@ -50,6 +50,9 @@ ditto.mongodb {
     retryWrites = true
     retryWrites = ${?MONGO_DB_RETRY_WRITES}
 
+    useAwsIamRole = false
```

Comment:
please add a comment in the line before what this configuration is for .. (enabling MongoDB passwordless authentication via AWS IAM)